frontend/src/components/unreviewed-sightings-page/UnreviewedSightings.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useEffect, useState, useContext } from "react";
+import { Redirect } from "react-router-dom";
+import {
+  getAllPendingSightings,
+  confirmOrRejectSighting,
+  Sighting,
+} from "../../clients/apiClient";
+import { LoginContext } from "../login/LoginManager";
+import { UnreviewedSightingCard } from "./UnreviewedSightingCard";
+
+export type ConfirmationStatus = 1 | 2;
```

Comment:
As in an earlier comment, instead of using this, import the string type from `apiClients.ts`